modules/graphstudio/pages/build-graph-patterns/visual-pattern-examples.adoc

```diff
@@ -1,5 +1,8 @@
 = Visual Pattern Examples
 
+IMPORTANT: In our ongoing efforts to refine and optimize GraphStudio, Build Graph Patterns will begin deprecation as of v3.9.3 and will not be supported in future versions of GraphStudio.
+Please visit our Insights documentation on xref:3.7@insights:widgets:index.adoc[] for graph and results visualization.
```

Comment:
Taking something away from GraphStudio is not optimizing GraphStudio. We need to focus broader.
In our ongoing efforts to refine and improve our products and tools, 
we are focusing on Insights at the tool of choice for building queries visually.  
We therefore are deprecating GraphStudio's Build Graph Patterns (Visual Query Builder) as of v3.9.3.
It will be discontinued in a future version of GraphStudio.

Reply:
Also, the link to insights should not be version 3.7. I think you can just omit the version number.